integration-test/ipfs.test-integration.js

```diff
@@ -0,0 +1,67 @@
+import IPFS from '../src/lib/IPFS/ipfs';
+import { config as loadEnv } from 'dotenv';
+import path from 'path';
+import faker from 'faker';
+const file = path.join(__dirname, 'infura.png');
+const unexistingFile = path.join(__dirname, 'infura-fake.png');
+
+loadEnv();
+describe('E2E Test: IPFS Service', () => {
+  jest.setTimeout(120 * 1000);
+  let ipfs;
+  let data1;
+  let data2;
+  const projectId = process.env.INFURA_IPFS_PROJECT_ID;
+  const projectSecret = process.env.INFURA_IPFS_PROJECT_SECRET;
+  const ipfsUrl = process.env.INFURA_IPFS_ENDPOINT;
+
+  beforeAll(async () => {
+    ipfs = new IPFS({ projectId, projectSecret, ipfsUrl });
```

Comment:
why not put `process.env.*` vars directly here instead of assigning to new vars?